tests/bin/pack-gear-tests.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+"""For directory/ies in the current directory zip into a test
+.zip archive with same name.
+
+This will REMOVE the test .zip file if it already exists and it
+will remove the test directory after zipping.
+
+Use "pack-tests.py all" to zip every test unless it is already
+unzipped.
+
+Example:
+    pack-tests.py hello_world
+
+"""
+
+import argparse
+import glob
+import os
+import shutil
+from pathlib import Path
+from zipfile import ZIP_DEFLATED, ZipFile
+
+
+def main():
+
+    exit_code = 0
+
+    # This can be run from:
```

Comment:
This  is very cool, but I'm not sure it's necessary unless we want to make it part of the template??

Reply:
The "pack" and "unpack" commands are part of the bids-app-template.  They allow easy editing/packaging of the gear tests as described [here](https://github.com/flywheel-apps/bids-app-template/blob/master/README.md).

Reply:
Oh, do you mean checking to see if it is in the proper place?

Reply:
Yea what I meant was checking to see where the script is running from.  It seems easier to just put a line in the readme/developer notes to say 'The tests need to be run from this dir...'

Reply:
Yeah, it's not really necessary.  I always run the pack/unpack commands from the `gear_tests`.